bigbluebutton-html5/imports/utils/deviceInfo.js

```diff
@@ -27,6 +27,7 @@ const deviceInfo = {
       isWindows: window.navigator.userAgent.indexOf('Windows') !== -1,
       isMac: window.navigator.userAgent.indexOf('Mac') !== -1,
       isLinux: window.navigator.userAgent.indexOf('Linux') !== -1,
+      isIOSChrome: navigator.userAgent.match('CriOS'),
```

Comment:
I believe so, as this fix does detect if the user is in the proper safari browser, because "safari = /safari/.test( userAgent )" returns true for both safari and iOSChrome, but this check allows the two browsers to be differentiated.